pkg/cli/admin/release/extract_tools.go

```diff
@@ -457,7 +457,10 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		if len(missing) == 1 {
 			return fmt.Errorf("the image %q containing the desired command is not available", missing.List()[0])
 		}
-		return fmt.Errorf("some required images are missing: %s", strings.Join(missing.List(), ", "))
+		if len(missing) > 1 {
+			return fmt.Errorf("some required images are missing: %s", strings.Join(missing.List(), ", "))
+		}
+		return fmt.Errorf("the command '%s' is not available for %s/%s on %s", command, runtime.GOOS, runtime.GOARCH, o.From)
```

Comment:
If the user passes `--command-os` flag, this message seems to be incorrect. It would be better to use `currentOS` and `currentArch`(set in here https://github.com/openshift/oc/blob/29cbcfb324607bb78c633ec46a54e8bac6576e82/pkg/cli/admin/release/extract_tools.go#L329) instead of runtime.

Reply:
Ok I'll set it to that instead

Reply:
Ok it's been updated